Workbooks/Windows Virtual Desktop/AtScale/Connection Reliability/Connection Reliability.workbook

```diff
@@ -478,7 +478,7 @@

```

Comment:
nit: the pattern to name columns "_real" could be confused with the schema disambiguation that Kusto uses for colliding column names with different types. I recommend "ClientTypeFull", "ClientIPAddressFull", "SessionHostIPAddressFull" instead (or *Unmasked for the IPs, etc.)

Reply:
Updated the variable name.